src/components/ChatList/elements/ChannelImage.tsx

```diff
@@ -64,7 +63,7 @@ const Big: React.FC<ChannelImageMainProps> = ({type, image, style}) => {
         style={[
           styles.containerImageGroupINfo,
           styles.containerDefaultImage,
-          styles.containerBigDarkBlue
+          styles.containerDarkBlue
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Again, the style `styles.containerDarkBlue` has been used instead of `styles.containerBigDarkBlue`. Make sure this change is intentional and does not negatively impact the UI.
```diff
- styles.containerDarkBlue
+ styles.containerBigDarkBlue
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->